src/EpiCategories/CategoryData.cs

```diff
@@ -8,6 +8,7 @@
 namespace Geta.EpiCategories
 {
     [AvailableContentTypes(Availability = Availability.Specific, Include = new[] { typeof(CategoryData) })]
+    [ContentType]
```

Comment:
```suggestion
    [ContentType(GUID = "9eba3e01-21b3-416e-b335-4911b07cf106", AvailableInEditMode = false)]
```

Reply:
I added a GUID to the ContentType and also set AvailableInEditMode to false since the intention is that you should create a content type that inherits from CategoryData class: https://github.com/Geta/EpiCategories#how-to-use

Reply:
I marked the conversation as resolved, but wanted to follow up on this. I agree with the change and think it makes sense.